packages/sample-app/sample_app/watsonx_flow.py

```diff
@@ -0,0 +1,62 @@
+import os
+import types
+from ibm_watsonx_ai.metanames import GenTextParamsMetaNames
+from ibm_watsonx_ai.foundation_models import ModelInference
+from pprint import pprint
+from traceloop.sdk import Traceloop
+from traceloop.sdk.decorators import workflow
+
+from dotenv import load_dotenv
+load_dotenv()
+
+os.environ['OTEL_EXPORTER_OTLP_INSECURE'] = 'True'
```

Comment:
Should this be documented as well if it's needed?

Reply:
Yes, sure, I will document this in https://github.com/traceloop/openllmetry/tree/main/packages/opentelemetry-instrumentation-watsonx